### PR TITLE
hotfix/missing dispatch in module ducks

### PR DESCRIPTION
**Pull request by BUGBOUNTYchrisg8691**
# What's here (& why)

- (include link(s) to applicable Trello card(s))

# Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete / work-in-progress, PR is for discussion or feedback
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How has this been tested?

- [ ] Tested locally
- [ ] Test suite provides the same results as prior to making changes

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/Modules/AddModuleForm.js [hunk 1]
```diff
@@ -39,23 +42,24 @@ const initialValues = {
...
+  const status = useSelector(state => state.moduleReducer.status);
+  const error = useSelector(state => state.moduleReducer.error);
 
   const changeValues = e => {
     const { name, value } = e.target;
     onChange(name, value);
   };
 
   useEffect(() => {
+    console.log(status);
```
Comment (chazkiker2) [inline src/components/Modules/AddModuleForm.js]: remove `console.log()`

File: src/components/Modules/AddModuleForm.js [hunk 2]
```diff
@@ -39,23 +42,24 @@ const initialValues = {
...
+    console.log(resetValues);
```
Comment (chazkiker2) [inline src/components/Modules/AddModuleForm.js]: remove `console.log()`

File: src/components/Modules/AddModuleForm.js [hunk 3]
```diff
@@ -135,5 +139,6 @@ export default function AddModuleForm() {
         </div>
       </Form>
     </StyledContainer>
+    // this is a test
```
Comment (chazkiker2) [inline src/components/Modules/AddModuleForm.js]: this test has passed.

(but also maybe delete comment)

File: src/components/Wrapper/Wrapper.js [hunk 4]
```diff
@@ -25,7 +25,7 @@ export default ({ children, ...restProps }) => {
         <Navigation authService={authService} />
         <Content>
           {/* should render the role and path on the main content rendering area */}
-          <LocationInfo>
+          {/* <LocationInfo>
```
Comment (chazkiker2) [inline src/components/Wrapper/Wrapper.js]: are we commenting this out?? or was this left over from previous debugging session?

File: src/hooks/useFormWithErrors.js [hunk 5]
```diff
@@ -13,7 +13,8 @@ export const useFormWithErrors = (
 
   const [values, setValues] = useState(initialValues);
   const [errors, setErrors] = useState(initialFormErrors);
-  const [disabled, setDisabled] = useState(initialDisabled);
+  // const [disabled, setDisabled] = useState(initialDisabled);
+  let disabled = initialDisabled;
```
Comment (chazkiker2) [inline src/hooks/useFormWithErrors.js]: we should change this back to what it previously was.

Check branch `main` for correct file

File: src/hooks/useFormWithErrors.js [hunk 6]
```diff
@@ -31,11 +32,14 @@ export const useFormWithErrors = (
   const resetValues = () => {
     setValues(initialValues);
     setErrors(initialFormErrors || initialValues);
-    setDisabled(true);
+    // setDisabled(true);
```
Comment (chazkiker2) [inline src/hooks/useFormWithErrors.js]: let's `setDisabled(initialDisabled)`

File: src/hooks/useFormWithErrors.js [hunk 7]
```diff
@@ -31,11 +32,14 @@ export const useFormWithErrors = (
...
+    disabled = initialDisabled;
   };
 
   useEffect(() => {
-    schema.isValid(values).then(valid => setDisabled(!valid));
+    // schema.isValid(values).then(valid => setDisabled(!valid));
```
Comment (chazkiker2) [inline src/hooks/useFormWithErrors.js]: take line 40 delete line 41

(use `setDisabled`)

File: src/state/ducks/moduleDuck.js [hunk 8]
```diff
@@ -88,6 +89,27 @@ export const moduleActions = {
...
+      thunkResolve,
+    } = moduleThunkUtils.getTriggersFromPrefix(dispatch, 'create');
+
+    thunkStart();
+
+    axiosAuth()
+      .post('/modules', newModule)
+      .then(res => {
+        dispatch({ status: CREATE_MODULE_SUCCESS, payload: res.data });
+        console.log(res.data);
```
Comment (chazkiker2) [inline src/state/ducks/moduleDuck.js]: remove `console.log`

**Review by chazkiker2 — CHANGES_REQUESTED** [review]
See comments above and let me know your thoughts. 

Also please remove `console.log()` statements and unnecessary comments. 

Overall this PR looks really really good. Thanks for the changes in the .md files, too.

File: src/components/Modules/AddModuleForm.js (hunk 2 above)
Comment (sebass482) [reply src/components/Modules/AddModuleForm.js]: fixed

File: src/components/Wrapper/Wrapper.js (hunk 4 above)
Comment (BUGBOUNTYchrisg8691) [reply src/components/Wrapper/Wrapper.js]: we are uncommenting this, it seems to be working correctly

File: src/components/Modules/AddModuleForm.js (hunk 1 above)
Comment (BUGBOUNTYchrisg8691) [reply src/components/Modules/AddModuleForm.js]: fixed

File: src/hooks/useFormWithErrors.js (hunk 5 above)
Comment (BUGBOUNTYchrisg8691) [reply src/hooks/useFormWithErrors.js]: fixed

File: src/hooks/useFormWithErrors.js (hunk 6 above)
Comment (BUGBOUNTYchrisg8691) [reply src/hooks/useFormWithErrors.js]: fixed

File: src/state/ducks/moduleDuck.js (hunk 8 above)
Comment (BUGBOUNTYchrisg8691) [reply src/state/ducks/moduleDuck.js]: fixed, I'm so sorry about all these. Thank you for catching them

**Review by chazkiker2 — APPROVED** [review]
Looks fantastic. Beautiful code, as always, sir

File: bw-prep.js [hunk 9]
```diff
@@ -58,7 +58,7 @@ const YAML = require('yaml');
   // remove scaffold .git 😈
   fs.rmdirSync('./.git', { recursive: true });
   // copy the .env.sample file to .env
-  fs.copyFile('./.env.sample', './.env', (err) => {
+  fs.copyFile('./.env.sample', './.env', err => {
     if (err) throw err;
     console.log('.env.sample was copied to .env');
```
Comment (sebass482) [inline bw-prep.js:63]: only console.log() I can find, although I don't think it should be removed

**Review by sebass482 — APPROVED** [review]
Changes to .md files and the moduleDuck are good. No lingering console-logs and passing all checks

**Pull request merged**